src/routes/safe/components/Settings/Advanced/RemoveModuleModal.tsx

```diff
@@ -50,7 +50,8 @@ const RemoveModuleModal = ({ onClose, selectedModule }: RemoveModuleModalProps):
   const safeAddress = useSelector(safeParamAddressFromStateSelector)
   const dispatch = useDispatch()
 
-  const explorerInfo = getExplorerInfo(selectedModule[0])
```

Comment:
can we rename `selectedModule` to `selectedPairModule`? 
also, could it be possible to assign `const prevModule =  selectedModule[0]`? I think it will help to understand what selectedModule[0] is.

Reply:
the assignment to a variable was already done, but I'm going with the `selectedModulePair` variable name